ReactNativeApp/src/trustly.tsx

```diff
@@ -0,0 +1,21 @@
+export const widget = (data = {}) => `<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Trustly</title>
+  <script src="http://192.168.0.9:8000/start/scripts/pwmb.js?accessId=A48B73F694C4C8EE6306" type="text/javascript"></script>
```

Comment:
update base URL to `sandbox.trustly.com` and update `accessId` to "YOUR_ACCESS_ID"

Reply:
As we moved the React Native to trustly-react-native-example repo, we can just ignore this PR and consider the new one. New PR https://github.com/TrustlyInc/trustly-react-native-example/pull/1